acceptance-tests/support/steps/input-steps.ts

```diff
@@ -62,3 +62,9 @@ Then("the value of the text field {} should be {string}", async function (this:
     );
     assert.equal(elementAttributeValue, attributeValueToCheck);
 });
+
+When("they submit the value of {} {string} that {}", async function (fieldName, value, condition) {
+    console.log(JSON.stringify(condition));
```

Comment:
Remove this line (the console.log ), we do not need the values logged out, logging out is used for debugging.